### PR TITLE
validation caisse

### PR DESCRIPTION
**Pull request by merveille-nitcheu**

### CODE REVIEW

File: src/Components/AddProduct.tsx [hunk 1]
```diff
@@ -1,101 +1,274 @@
...
+  compte_tiers?: string;
+  libelle_ecriture?: string;
+  date_echeance?: string;
+  position_journal?: string;
+  debit?: string;
+  credit?: string;
+}
 
+export default function AddProduct(props: VisibilityProps) {
+  /* if (props.setProduct && props.selectedProduct) {
```
Comment (nidovic) [inline src/Components/AddProduct.tsx:30]: why you comment that instead remove ??

File: src/Components/ShowProduct.tsx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+  name?: string;
+  code: number | null;
+}
+
+const type: Type[] = [
+  { name: "Toutes les ecritures", code: null },
+  { name: "Ecritures lettrées", code: 1 },
+  { name: "Ecritures non lettrées", code: 2 },
+];
+const produit: Product[] = [
```
Comment (nidovic) [inline src/Components/ShowProduct.tsx:47]: why you don't make that in file call like database/fake_data.tsx

File: src/Components/ShowProduct.tsx [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+      setShowProduct(_product);
+    }
+  }, [products]);
+
+  const [filters, setFilters] = useState<DataTableFilterMeta>({
+    type: { value: null, matchMode: FilterMatchMode.CONTAINS },
+  });
+  const onGlobalFilterChange = (e: DropdownChangeEvent) => {
+    const { code } = e.target.value;
+    let _filters = { ...filters };
```
Comment (nidovic) [inline src/Components/ShowProduct.tsx:167]: any reason for declare that with let instead of const ..

File: src/Components/ShowProduct.tsx [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+      <div className="flex flex-wrap gap-5">
+        <SplitButton
+          label="Actions"
+          icon="pi pi-cog"
+          model={items}
+          size="small"
+          className="buton"
+          style={{ backgroundColor: "#3b82f6" }}
+        />
+        {/* <Button
```
Comment (nidovic) [inline src/Components/ShowProduct.tsx:186]: remove instead comment.

File: src/Components/ValidationCaisse.tsx [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+  { name: "Atabe", code: 1 },
+  { name: "SIAKAM", code: 2 },
+];
+
+const notif: Type[] = [
+  { name: "Aucun", code: null },
+  { name: "Mail", code: 2 },
+  { name: "Sms", code: 3 },
+];
+const produit: Product[] = [
```
Comment (nidovic) [inline src/Components/ValidationCaisse.tsx:70]: why you don't make that in file call like database/fake_data.tsx in order to easily reuse that.

File: src/index.css [hunk 6]
```diff
@@ -28,54 +27,165 @@ code {
...
+  font-size: small;
+  margin-top: 20px;
+}
+
+.p-confirm-dialog-reject {
+  margin-left: 15px;
+  height: 35px;
+  font-size: small;
+}
+/* 768 */
```
Comment (nidovic) [inline src/index.css:125]: why that ?

**Review by nidovic — APPROVED** [review]
organize this project like : src/pages/name_of_page/components   and src/components/name _of_components

File: src/Components/AddProduct.tsx (hunk 1 above)
Comment (merveille-nitcheu) [reply src/Components/AddProduct.tsx:30]: because I'm not completely finished with this feature yet

File: src/Components/ShowProduct.tsx (hunk 2 above)
Comment (merveille-nitcheu) [reply src/Components/ShowProduct.tsx:47]: I hadn't thought about it, I would do it

File: src/Components/ShowProduct.tsx (hunk 3 above)
Comment (merveille-nitcheu) [reply src/Components/ShowProduct.tsx:167]: allright

File: src/Components/ShowProduct.tsx (hunk 4 above)
Comment (merveille-nitcheu) [reply src/Components/ShowProduct.tsx:186]: ok

File: src/Components/ValidationCaisse.tsx (hunk 5 above)
Comment (merveille-nitcheu) [reply src/Components/ValidationCaisse.tsx:70]: it's planned

File: src/index.css (hunk 6 above)
Comment (merveille-nitcheu) [reply src/index.css:125]: when I do my tests, I sometimes forget to remove my comments

**Pull request merged**